lib/platform/github/index.ts

```diff
@@ -1706,6 +1706,12 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
       if (alerts.length) {
         logger.trace({ alerts }, 'GitHub vulnerability details');
         for (const alert of alerts) {
+          if (!alert.securityVulnerability) {
```

Comment:
```suggestion
          if (alert.securityVulnerability === null) {
```
We should be explicit here. `undefined` should still be an error.

Reply:
@viceice Thank you for your suggestion. I commit it.